assets/src/apps/authoring/components/Flowchart/screens/screen-factories.ts

```diff
@@ -16,6 +16,19 @@ export const screenTypes = [
   'dropdown',
 ];
 
+export const screenType = {
```

Comment:
I think if you are going to create an enum for the screenType then it should be an actual typescript string enum, and also you should use it again in the other items here, like the above screenTypes would then be like:
```
export const screenTypes = [
  screenType.BLANK_SCREEN,
  screenType.WELCOME_SCREEN,
  // ... etc
  ```